src/pages/DragonPage/DargonAlert.tsx

```diff
@@ -0,0 +1,39 @@
+import React, { useState } from 'react';
+import { Box, Typography, useTheme } from '@material-ui/core';
+import { ReactComponent as AlertIcon } from 'assets/images/AlertIcon.svg';
+import { ReactComponent as CloseIcon } from 'assets/images/CloseIcon.svg';
+
+const DragonAlert: React.FC = () => {
```

Comment:
Rename the file it's spelled wrong

Reply:
change `DargonAlert.tsx` to `DragonAlert.tsx` and pushed update